ruby-app-onbuild/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM ministryofjustice/ruby:2.1.5
+FROM ministryofjustice/ruby:2
```

Comment:
So this depends on how we are tagging our ministryofjustice/ruby image.

I think this was being done automatically via Dockerhub but I'm not certain.... this but might need looking at.